WcaOnRails/app/views/posts/_post.html.erb

```diff
@@ -22,6 +22,6 @@
     <div>
       <em>Announced by <%= post.author.name %> on <%= local_time(post.created_at) %></em>
     </div>
-    <%= post.body.html_safe %>
```

Comment:
Just ran this script on the live site and it worked perfectly =). Thanks again, @timhabermaas!